fhservices.cfg.j2

```diff
@@ -184,6 +201,33 @@ define service {
 }
 {% endif %}
 
+define service {
+       service_description jenkins:ping
+       check_command check_jenkins!80!/login
+       use generic-service
+       hostgroup_name digger
+       contact_groups rhmapadmins
+}
+
+define service {
+       service_description mac_ios:ping
+       check_command check_ios_machines!{{ jenkins_host }}!443!{{ jenkins_user }}!{{ jenkins_pass }}
```

Comment:
The issue with these is that the env variables are pulled into the make script but never pushed into the commands.cfg partial file which eventually gets merged to /etc/nagios/conf.d/fhservices.cfg.